pkgs/applications/video/obs-studio/plugins/obs-livesplit-one.nix

```diff
@@ -0,0 +1,37 @@
+{ lib
+, fetchFromGitHub
+, rustPlatform
+, cmake
+, fontconfig
+, obs-studio
+, pkg-config
+}:
+
+rustPlatform.buildRustPackage rec {
```

Comment:
Move this file to `pkgs/applications/video/obs-studio/plugins/obs-livesplit-one/default.nix` (create the `obs-livesplit-one` folder).

This makes it easier to add patches later.